util/chunk/disk.go

```diff
@@ -378,35 +387,28 @@ func convertFromRow(row Row, reuse *diskFormatRow) (format *diskFormatRow) {
 	return
 }
 
-// toMutRow deserializes diskFormatRow to MutRow.
-func (format *diskFormatRow) toMutRow(fields []*types.FieldType) MutRow {
-	chk := &Chunk{columns: make([]*Column, 0, len(format.sizesOfColumns))}
+// toRow deserializes diskFormatRow to Row.
+func (format *diskFormatRow) toRow(fields []*types.FieldType, chk *Chunk) (Row, *Chunk) {
+	if chk == nil || chk.IsFull() {
```

Comment:
Just a quesction, `chk == nil` maybe unnecessary? Because sync.Pool will always new a chunk.

Reply:
Check `chk==nil` is more safe.. For example, the code in test cases doesn't have the sync.Pool and doesn't need to reuse.